README.md

```diff
@@ -103,7 +135,20 @@ To define a token, simply invoke `morgan.token()` with the name and a callback f
 morgan.token('type', function (req, res) { return req.headers['content-type'] })
 ```
 
-Calling `morgan.token()` using the same name as an existing token will overwrite that token definition.
+There are a few pieces of behavior you should be aware of when using tokens:
+
+- `morgan.token('name', callback)` corresponds to `:name` in any string passed to `morgan(format, options)`.
```

Comment:
Sure. Changed this in [1c9416c](https://github.com/expressjs/morgan/pull/125/commits/1c9416c7692b359b2f4bcfe7b9b357fe14dc9306). Not sure why this Github comment wasn't hidden automatically because the line was changed.